spark_parser/spark.py

```diff
@@ -494,11 +500,16 @@ def makeSet(self, token, sets, i):
 
             if parent == i:
                 continue
-
+            if self.debug['reduce']: #STRANIKS_SCAN 01/03/2016
```

Comment:
Rather than comment on ever single line that you change, I'd prefer one note somewhere. 

The git history records who changed which line specifically. And it picks up the date and time automatically too!